share/wrf_timeseries.F

```diff
@@ -379,6 +399,7 @@ SUBROUTINE calc_ts( grid )
             DO k=1,grid%max_ts_level
             grid%ts_u_profile(n,i,k)   = earth_u_profile(k)
             grid%ts_v_profile(n,i,k)   = earth_v_profile(k)
+            grid%ts_w_profile(n,i,k)   = (grid%w_2(ix,k,iy)+grid%w_2(ix,k+1,iy))/2.0 ! w on cell center
             grid%ts_gph_profile(n,i,k) = (grid%phb(ix,k,iy)+grid%ph_2(ix,k,iy))/9.81 
             grid%ts_th_profile(n,i,k)  = grid%t_2(ix,k,iy) + 300
```

Comment:
@phawbeck @mgduda @kkeene44 
The grid%t_2 variable is likely to be a moist potential temperature perturbation (based on a nml setting).
t_2 = ( full dry theta ) * ( 1 + Rv/Rd Qv ) - 300
Should I fix this in a separate PR, or address this theta inconsistency in this PR?

Reply:
It is possible that the sensible temperature (if T is based on an assumed dry potential temperature) is incorrect also, but I have not looked at this code thoroughly.

Reply:
@phawbeck @kkeene44 @mgduda 
Folks,
There is likely a bug in the treatment of potential temperature for the tslist option, but there is likely this exact same bug in several other diagnostic routines (with the introduction of moist theta in v4.0). We will not deal with the dry potential temperature issue in this PR.